README.md

```diff
@@ -9,17 +9,17 @@ An apt based linux system
 
 ## Role Variables
 
-
 | Variable                | Default / Mandatory                                             | Description                                                                                                                                                                                                                                              |
 |-------------------------|-----------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `logstash_config`       | `{ path: { logs: /var/log/logstash, data: /var/lib/logstash }}` | Since the logstash settings file is a yaml file we write the whole object to the settings file. For more Information please see the [logstash settings file documentation](https://www.elastic.co/guide/en/logstash/current/logstash-settings-file.html) |
 | `logstash_config_files` | `{}`                                                            | Dictionary of config files to write to conf.d/. More information below.                                                                                                                                                                                  |
 
 ### `logstash_config_files`
-Each entry in the `logstash_config_files` consits out of the following entries.
+Each entry in the `logstash_config_files` consists out of the following entries.
```

Comment:
Each entry in `logstash_config_files` is also a dict with the following keys:

(I don't know how it would be possible to define not-unique entries in a dict :P

Reply:
fixed